website/views/shared/profiles/achievements.jade

```diff
@@ -78,9 +78,9 @@ div(ng-if='::profile.achievements.perfect || user._id == profile._id')
     small(ng-if='::profile.achievements.perfect == 1')=env.t('perfectSingularText')
   hr
 
-div(ng-if='::profile.achievements.ultimateGearSets || user._id == profile._id')
-  .achievement.achievement-armor(ng-if='::profile.achievements.ultimateGearSets')
-  div(ng-class='::{muted: !profile.achievements.ultimateGearSets}')
+div(ng-if='(user._id == profile._id) || profile.achievements.ultimateGearSets.healer || profile.achievements.ultimateGearSets.wizard || profile.achievements.ultimateGearSets.rogue || profile.achievements.ultimateGearSets.warrior')
```

Comment:
Would the profile.achievements check be worth putting into a function? Even saving them in a jade variable would be better and more DRY.

Any reason to remove the `::` binding?